skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +355,136 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
+    def test_ancom_basic_counts(self):
+        test_table = pd.DataFrame(self.table1)
+        original_table = copy.deepcopy(test_table)
+        result = ancom(test_table,
+                       pd.Series(self.cats1),
+                       multiple_comparisons_correction=None)
+
+        # Test to make sure that the input table hasn't be altered
+        npt.assert_allclose(original_table, test_table)
```

Comment:
`assert_data_frame_almost_equal(original_table, test_table)`

Also check that `grouping` isn't modified in-place